pennylane/templates/state_preparations/mottonen.py

```diff
@@ -76,7 +76,8 @@ def _matrix_M_entry(row, col):
 
 
 def _compute_theta(alpha):
-    """Calculates the rotation angles from the alpha vector.
+    """Maps the angles alpha of the multi-controlled rotations decomposition of a uniformly-controlled rotation
```

Comment:
```suggestion
    """Maps the angles alpha of the multi-controlled rotations decomposition of a uniformly controlled rotation
```

Original work uses "uniformly controlled" without the hyphen, I'm marking only this instance but I noticed a couple of times the hyphen appears throughout.